### PR TITLE
Fix export csv function #363

### PR DESCRIPTION
**Pull request by jefmoura**
## Purpose
The export functionality wasn't working properly. The users should be able to export data to CSV.

### CODE REVIEW

File: silo/tests/test_views.py [hunk 1]
```diff
@@ -138,3 +147,74 @@ def test_index_get_unauthenticated_no_activity_url(self):
...
+    def tearDown(self):
+        if self.silo_id:
+            # Have to remove the created lvs
+            lvss = LabelValueStore.objects.filter(silo_id=self.silo_id)
+            for lvs in lvss:
+                lvs.delete()
+
+    def test_export_csv(self):
+        self.tola_user.user.is_staff = True
+        self.tola_user.user.is_superuser = True
```
Comment (Menda) [inline silo/tests/test_views.py]: Do we need to be superuser for this?

Comment (jefmoura) [reply silo/tests/test_views.py]: Doesn't matter. I'm gonna create another test with a normal user.

File: silo/tests/test_views.py [hunk 2]
```diff
@@ -138,3 +147,131 @@ def test_index_get_unauthenticated_no_activity_url(self):
...
+        request.user = self.tola_user.user
+        response = views.export_silo(request, self.silo_id)
+        self.assertEqual(response.status_code, 200)
+        self.assertIn('color,type', response.content)
+        self.assertIn('black,primary', response.content)
+
+    def test_export_csv_normaluser(self):
+        # The user has to be a superuser to create a custom form
+        user = factories.User(first_name='John', last_name='Lennon',
+                              is_staff=True, is_superuser=True)
```
Comment (Menda) [inline silo/tests/test_views.py]: This is wrong for a normal user

**Comment by Menda** [comment]
I don´t think we need tests for normal and superuser, as there is no distinction inside the export logic

File: silo/tests/test_views.py (hunk 2 above)
Comment (jefmoura) [reply silo/tests/test_views.py]: Double check the users of the requests. It's not wrong.

Comment (jefmoura) [reply silo/tests/test_views.py]: As I said, to create a custom form you need to be a superuser.

**Comment by jefmoura** [comment]
In the end, we don't need the normal user test, so I removed it. @Menda

**Pull request merged**